cross/libwebsockets/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = libwebsockets
 PKG_VERS = 1.4-chrome43-firefox-36
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://git.libwebsockets.org/cgi-bin/cgit/libwebsockets/snapshot
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/warmcat/libwebsockets/archive
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

Comment:
@cytec, think it's `PKG_DIST_FILE`. Having said that, it's probably a good idea to verify which version of libwebsockets to use. According to the changelog, Mosquitto should be able to support a newer version than 1.4 (1.6, it looks like)